src/app/admin/platform-config/platform-config.component.html

```diff
@@ -26,41 +36,67 @@ <h1 class="app-page-title">PLATFORM CONFIGURATION</h1>
                                  [firstFileControl]="updateForm.get('logo')">
                 </app-upload-area>
 
-                <label for="icon" class="app-page-label mt-3">Icon</label>
+                <label for="icon" class="app-page-label mt-3">
+                    {{ 'admin.platform_config.icon_url_input.title' | translate }}
+                </label>
                 <input class="input-reg w-100" id="icon" formControlName="icon"
-                       placeholder="https://path/to/icon.png" type="text">
+                       placeholder="{{ 'admin.platform_config.icon_url_input.placeholder' | translate }}"
+                       type="text">
+
+                <label for="coop-statute" class="app-page-label mt-3">
+                    {{ 'admin.platform_config.coop_statute_input.title' | translate }}
+                </label>
+                <input class="input-reg w-100" id="coop-statute" formControlName="coop_statute"
+                       placeholder="{{ 'admin.platform_config.coop_statute_input.placeholder' | translate }}"
+                       type="text">
 
                 <!-- Hidden because of security reasons. -->
-                <div [hidden]="true">
-                    <label for="arkane-id" class="app-page-label mt-3">Arkane ID</label>
+                <div [hidden]="!showSecureConfig">
+                    <label for="arkane-id" class="app-page-label mt-3">
+                        {{ 'admin.platform_config.arkane_id_input.title' | translate }}
+                    </label>
                     <input class="form-control p-4" id="arkane-id" formControlName="arkaneID"
-                           placeholder="arkaneID" type="text">
+                           placeholder="{{ 'admin.platform_config.arkane_id_input.placeholder' | translate }}"
+                           type="text">
```

Comment:
Can you please change input class to `input-reg` instead of `form-control` on inputs in this section? I didn't do it because it used to be hidden... My bad

Reply:
ofc!